### PR TITLE
Update discover home

### PR DESCRIPTION
**Pull request by benisgold**
Fixes APP-207

## What changed (plus any additional context for devs)
- updated layout of discover home w/ new cards

## Screen recordings / screenshots
Simulator Screen Shot - iPhone 14 Pro Max - 2022-11-10 at 14 36 28
Simulator Screen Shot - iPhone 14 Pro Max - 2022-11-10 at 14 39 13

### CODE REVIEW

File: src/components/cards/utils/constants.ts [hunk 1]
```diff
@@ -108,41 +108,48 @@ export const getLearnCardColorway: (
...
 
-const avoidScamsCard: LearnCardDetails = {
+export const avoidScamsCard: LearnCardDetails = {
   url: 'https://learn.rainbow.me/avoid-crypto-scams',
   category: LearnCategory.Safety,
   key: 'avoid_scams',
   emoji: '🤬',
 };
 
+export const cryptoAndWalletsCard: LearnCardDetails = {
```
Comment (benisgold) [inline src/components/cards/utils/constants.ts:146]: will possibly refactor / get rid of this file when i make the wallet screen changes

**Review by skylarbarrera — APPROVED** [review]
🚀

**Comment by skylarbarrera** [comment]
looks like this needs some e2e magic

**Review by jkadamczyk — CHANGES_REQUESTED** [review]
I reviewed the code and it looks grweat, I also did some dev QA, I shared a loom recording on Linear

File: src/components/cards/LearnCard.tsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ export const LEARN_CARD_HEIGHT = 184;
 
 type LearnCardProps = {
   cardDetails: LearnCardDetails;
-  type: 'square' | 'stretch';
+  type: CardType;
```
Comment (estrattonbailey) [inline src/components/cards/LearnCard.tsx:18]: Nice!

**Review by estrattonbailey — APPROVED** [review]

File: src/components/cards/LearnCard.tsx (hunk 2 above)
Comment (benisgold) [reply src/components/cards/LearnCard.tsx:18]: haha thank u

**Comment by benisgold** [comment]
@jkadamczyk i fixed all the things in your loom vid! thanks for noting those

**Review by jkadamczyk — APPROVED** [review]
Looks great!

Very smart with pointerEvents="none" 👍

**Pull request merged**